cluster_test.go

```diff
@@ -19,12 +19,12 @@ var slotsResp = newResult(RedisMessage{typ: '*', values: []RedisMessage{
 		{typ: ':', integer: 0},
 		{typ: ':', integer: 16383},
 		{typ: '*', values: []RedisMessage{ // master
-			{typ: '+', string: ""},
+			{typ: '+', string: "127.0.0.1"},
```

Comment:
@rueian I choose to put IP everywhere since it was the default behaviour you had in mind right?
And I created a new slot response without the IP set: `singleSlotWithoutIP`

Reply:
Yes, many thanks!